### PR TITLE
changed formatting

### PR DESCRIPTION
**Pull request by sartechb**
Changed format to be more in line with what @themoh needed for #67 and #68 (and #81).

### CODE REVIEW

**Comment by the-alex** [comment]
This is a small thing to ask, but the preview (section above ) contains multiple usage examples in the code block section, making it a longer than the other command previews. Could you break it up so that only one usage example is in that first code block is visible from the main "reference" page? The rest can just go after in the "Useful Options / Examples" section.

Otherwise, everything else looks good.

**Comment by sartechb** [comment]
@the-alex done! How's that look?

File: _commands/os_x/open.md [hunk 1]
```diff
@@ -35,11 +31,35 @@ open -a Safari http://www.google.com
...
+| -R, --reveal | Selects in the Finder instead of opening |
+|  -n, --new   | Opens a new instance of the application (even if it's already running) |
+|  -j, --hide  |         Launches the app hidden          |
+
 
-As we can see, open is fairly easy to use. For more flags, such as launching an application as hidden, please check out the man page.
+#### `open -a Safari --hide http://www.google.com`
+Opens google.com in Safari minimized by default
+##### Break it down
+The `-a` flag lets you choose an application to open your item with. Without it, the item would open up with the default handler. For example, if you remov `-a Safari` and your default browser is Chrome then google.com will open in Chrome.
```
Comment (themoh) [inline _commands/os_x/open.md]: This is what this looks like: 
screen shot 2016-03-19 at 3 14 31 pm

Fix formatting.

Comment (sartechb) [inline _commands/os_x/open.md]: @themo 
I'm not seeing this issue. I'll add a newline though and see if it fixes the problem you're seeing.
image

File: _commands/os_x/open.md [hunk 2]
```diff
@@ -1,27 +1,23 @@
...
-|     Flag     |               Description                |
-| :----------: | :--------------------------------------: |
-|      -a      |  Opens item with a specific application  |
-|      -e      |         Opens item with TextEdit         |
-|      -t      |   Opens item with default text editor    |
-|      -f      | Reads input from standard input and opens it with TextEdit |
-| -R, --reveal | Selects in the Finder instead of opening |
-|  -n, --new   | Opens a new instance of the application (even if it's already running) |
-|  -j, --hide  |         Launches the app hidden          |
+OPEN (OSX)
```
Comment (themoh) [inline _commands/os_x/open.md]: lowercase all text and remove '(OS)' completely, its already in the section name

File: _commands/os_x/open.md [hunk 3]
```diff
@@ -1,27 +1,23 @@
...
-####Usage
+`open` is used to open files, directories or URLs from the terminal. 
 
-~~~bash
+~~~ bash
 #Opens google.com in your default browser
 open http://www.google.com
+~~~
 
+For more advanced commands please check out the `man` page of open: `man open`.
```
Comment (themoh) [inline _commands/os_x/open.md]: Remove this line completely.

**Comment by themoh** [comment]
Please check your spelling and grammar as well.

**Comment by sartechb** [comment]
@themoh I think that's everything!

**Comment by the-alex** [comment]
Nothing renders incorrectly on my machine.

**Comment by themoh** [comment]
Looks a lot better now. Good work!

**Pull request merged**